frontend/src/Editor/Components/TextInput.jsx

```diff
@@ -98,7 +98,7 @@ export const TextInput = function TextInput({
           darkMode && 'dark-theme-placeholder'
         }`}
         placeholder={properties.placeholder}
-        style={{ height, borderRadius: `${styles.borderRadius}px`, color: textColor }}
+        style={{ height, borderRadius: `${styles.borderRadius}px`, color: textColor, backgroundColor: `${styles.backgroundColor}`}}
```

Comment:
Also need to handle the background color for the default value on toggling the theme